Sources/Commands/SwiftTestCommand.swift

```diff
@@ -80,6 +80,38 @@ struct SharedOptions: ParsableArguments {
     var testProduct: String?
 }
 
+struct TestEventStreamOptions: ParsableArguments {
+    /// Legacy equivalent of ``configurationPath``.
+    @Option(name: .customLong("experimental-configuration-path"),
+            help: .private)
```

Comment:
What's private vs hidden? Haven't seen private before 🤔

Reply:
Hidden means it still shows up in verbose help. Private means it is never shown in help.